.github/CONTRIBUTING.rst

```diff
@@ -50,7 +50,7 @@ available for review as a PDF.
 
 NOTE: If you work for a company that wants to allow you to contribute your work,
 then you'll need to sign a `corporate CLA <https://qiskit.org/license/qiskit-corporate-cla.pdf>`_
-and email it to us at qiskit@us.ibm.com.
+and email it to us at qiskit.terra@us.ibm.com.
```

Comment:
should we have different emails. I think a single email is enough

Reply:
yes! this is a mistake.